### ISSUE TITLE
Catching errors thrown by Utils.throwError

### ISSUE THREAD

**Issue by findel** [ShyykoSerhiy/skyweb#24]
Hey!

Question (please forgive me if I'm not seeing something obvious, I'm quite new to node):

When an error is handled using Utils.throwError is doesn't actually throw the error.

https://github.com/ShyykoSerhiy/skyweb/blob/master/dist/utils.js#L8

How should I 'catch' these errors so that I can handle them correctly?

Phil

**Comment by ShyykoSerhiy** [comment]
Yeah, need to rewrite to add a way to actually respond to this errors.

**Comment by ShyykoSerhiy** [comment]
parially fixed in 66ab6476cf3d499c5d0ef435e8cb26e76decbefc
